engines/sci/sound/music.cpp

```diff
@@ -261,10 +261,17 @@ void SciMusic::clearPlayList() {
 
 void SciMusic::pauseAll(bool pause) {
 	const MusicList::iterator end = _playList.end();
+	bool alreadyUnpaused = (_globalPause <= 0);
+
 	if (pause)
 		_globalPause++;
 	else
-		_globalPause = MAX<int>(_globalPause - 1, 0);
+		_globalPause--;
+
+	bool stillUnpaused = (_globalPause <= 0);
+	if (alreadyUnpaused && stillUnpaused)
```

Comment:
Don't we need to check for the pause flag here?
```suggestion
	if (alreadyUnpaused && stillUnpaused && pause)
```

Reply:
No, if you do that it will become "assymetric" again and glitch the same way as described in the bug ticket. I've just tested it to make sure. I have added a comment to the PR which will hopefully make this a bit more transparent...

Reply:
Thanks for clearing that out